app/scripts/views/app.coffee

```diff
@@ -569,10 +564,22 @@ define [
       # TODO @jrwdunham: use the already-in-place @alertDialog for this
       # TODO @jrwdunham @cesine: figure out how i18n/localization works in
       # FieldDB and begin implementing something similar in Dative.
-      console.log ["TODO show some visual thing here using the app view using",
+      console.log ["Pretending the user said Yes. This is dangerous. TODO show some visual confirm dialog here using the app view using",
         "something like http://jqueryui.com/dialog/#modal-confirmation" ,
         message].join ' '
 
+      # NOTE cesine: overiding confirm MUST return a promise, otherwise its broken.
+      deferred = FieldDB.Q.defer()
+      self = this
```

Comment:
In general, you shouldn't need to write `self = this` in CoffeeScript. Anyways, you're not even using `self` here ...